apiary.apib

```diff
@@ -3374,6 +3374,12 @@ Add a draft work order.
                + product_id: `e2314517-3cab-4aa9-8471-450e73449040` (string, required)
                + quantity: 3 (number, required)
        + location (Address, optional)
+       + work_hours (array, optional)
+           + (object)
+               + user_id: `4ec55a8c-2d80-472a-a9c2-5ff5ee7eac6a` (string, required)
+               + started_at: `2019-02-27T10:01:49+00:00` (string, required)
+               + duration (Duration, required)
```

Comment:
Q: Should we have an end time instead? It's easier to say when you started & ended than to calculate the amount of time you worked.

Reply:
We checked this with the mobile team and they prefer this format (I guess this is what they've been using already for time tracking). Since we need to move fast with WorkOrders we're just accepting this format for now.

Reply:
If we want to offer the same functionality as time tracking can we actually make it the same? Because now we're introducing a duration object while time tracking just has a duration in seconds.

My PP would be to offer options with duration and start/end like `timeTracking.add`.

Reply:
We will keep it in sync with the time tracking options.
So you can provide duration as an integer. 
Later on, we can add the start/end time option.

Reply:
For TimeTracking it is indeed just duration in seconds, I don't know why this is an Object here.
Basically I agree with Jens to make it the same as TimeTracking but for simplicity perhaps only do the started_at + duration combination instead of everything (started_at + ended_at, ended_at + duration, ...)

Reply:
Indeed. The object has been removed. And as discussed with mobile, we only implemented the first combination due to deadlines.